src/main.lua

```diff
@@ -729,14 +731,23 @@ addEventingHandler(ActionMap.VaultedTransfer, utils.hasMatchingTag("Action", Act
 	)
 	assert(recipient ~= msg.From, "Cannot transfer to self")
 
-	local vault =
-		vaults.vaultedTransfer(msg.From, recipient, quantity, lockLengthMs, msg.Timestamp, msgId, allowUnsafeAddresses)
+	local vault = vaults.vaultedTransfer(
+		msg.From,
```

Comment:
Make sure from and recipient are passed through normalized. My quick reading of the code is the choice of using msg.From and msg.Tags.Recipient should be normalized by the sanitization handler.